utils/src/command.rs

```diff
@@ -84,6 +88,19 @@ impl CommandRunExt for Command {
         self.status()?.check_status(stderr)
     }
 
+    #[allow(unsafe_code)]
+    fn lifecycle_bind(&mut self) -> &mut Self {
+        // SAFETY: This API is safe to call in a forked child.
```

Comment:
A docs reference would be good (even if it's just to say that this is implemented with prctl and point to the list in signal-safety manpage)...

Reply:
Hmm, the rustix docs for this function do link directly to the prctl man page right?

Reply:
I think while we *could* have more safety explanations in cases like this, what we have here is actually better than some of the other (few) `unsafe` cases and if we were to try to raise the bar we'd need to a bit more